lib/rubocop-rspec.rb

```diff
@@ -40,10 +39,13 @@
 require_relative 'rubocop/rspec/factory_bot'
 require_relative 'rubocop/rspec/hook'
 
-RuboCop::RSpec::Inject.defaults!
-
 require_relative 'rubocop/cop/rspec_cops'
 
+project_root = File.join(__dir__, '..')
+RuboCop::ConfigLoader.inject_defaults!(project_root)
```

Comment:
Why don’t we pass in the constant `RuboCop::RSpec::PROJECT_ROOT` here? Although, it’s marked as a private constant, so I don’t know if it’s accessible here.